src/constants.ts

```diff
@@ -1,2 +1,20 @@
 export const tfa_text =
   'A verification code has been sent to your email address. Please check your inbox and enter the code to complete the authentication process. Code Sent to';
+export const forgot_password = {
+  title: 'Forgot Password',
+  text: `To reset your password, please enter the email address associated with your account. We will send you a secure password reset link to that email.`,
+};
+export const reset_password_success = {
+  image: '../../../images/ic_outline-check-circle-outline.png',
+  text: `Password reset successfully!`,
+};
+
+export const resent_link_sent = {
+  image: '../../../images/sent.png',
```

Comment:
this kind of using images, or any other file from the public, is likely to cause problems during build, or development sometimes. Simply use `./images/**.png` , since you are inside `src`, this is the best way.  And this path  is not matching any file anyways: `'../../../images/sent.png'`